plugins/modules/cloud/packet/packet_device.py

```diff
@@ -424,6 +431,17 @@ def get_device_id_list(module):
     return device_ids
 
 
+def get_tags_list(module):
```

Comment:
What is this function needed for?

Reply:
That was reserved for future use (device operations based on tags). I removed for now it to avoid any confusion.

Reply:
It shouldn't be needed anyway, since you declared the module option as `list` with `str` elements, Ansible will make sure that you get a list of strings. If someone passes a string to the module, it will be split along `,` (and possibly spaces are removed) so that you end up with a list.